internal/cmd/grpc.go

```diff
@@ -361,6 +367,33 @@ func NewGRPCServer(
 	return server, nil
 }
 
+// NewObjectStore create a new storate.Store from the object config
+func NewObjectStore(cfg *config.Config, logger *zap.Logger) (storage.Store, error) {
+	objectCfg := cfg.Storage.Object
+	var store storage.Store
+	switch objectCfg.Type {
+	case config.S3ObjectSubStorageType:
```

Comment:
feel free to add https://golangci-lint.run/usage/false-positives/#nolint-directive above here, since we'll likely support more object storage backends going forward so the switch will be needed